### ISSUE TITLE
how do you download a specific file from an item?

### ISSUE THREAD

**Issue by ofsaleem** [nektro/go-internetarchive#4]
the archive.org docs for the python ia show that you can download a specific file by doing
`ia download item filename`

but when i try that with goia it just downloads the entire item (which contains many many files)

**Comment by nektro** [comment]
is there a use case where going directly to the website for this would not be ideal?

**Comment by ofsaleem** [comment]
i found your tool from a reddit thread where they suggested it was much faster than downloading via web, so i assume that would be why. using jdownloader i was able (since it opens multiple connections?) to get about 10x the rate i was getting via directly web, so i thought goia might give a speed boost too. but if not, then no, probably no use case :)

**Comment by nektro** [comment]
> since it opens multiple connections?

this is correct and where I presume most of the speedup is coming from along with IA using load balancers. so over a single file I don't think you'd see the same speedup. happy to be proved wrong though if anyone has seen otherwise

> i found your tool from a reddit thread

welcome! and happy to hear you've been having a good time with the tool 😄
